tests/test-keyring/meson.build

```diff
@@ -0,0 +1,23 @@
+foreach file : [
+  'README',
+  'pubring.gpg',
+  'secring.gpg',
+]
+  # TODO: When we depend on Meson 0.50.0, we can use
+  # install : get_option('installed_tests')
+  if get_option('installed_tests')
+    configure_file(
+      input : file,
+      output : file,
+      copy : true,
+      install_dir : installed_testdir / 'test-keyring',
+      install_mode : 'rw-r--r--',
+    )
+  else
+    configure_file(
+      input : file,
+      output : file,
+      copy : true,
+    )
+  endif
+endforeach
```

Comment:
You can pass `install_dir: false` here. Or rather:
```
if get_option('installed_tests')
    idir = installed_testdir / 'test-keyring'
else
    idir = false
endif

configure_file(
    input : file,
    output : file,
    copy : true,
    install_dir : idir,
    install_mode : 'rw-r--r--',
)
```

This works fine for now, and once you bump your minimum version, it will start saying this:

```
WARNING: Project targeting '>=0.50.0' but tried to use feature deprecated since '0.50.0': configure_file install_dir: false. Use the `install:` kwarg instead
```

Reply:
Benefit: less code to sync across multiple branches.